src/jmc/compile/command/utils.py

```diff
@@ -615,7 +615,6 @@ def __parse_bracket(self) -> None:
                 if isinstance(json_body, dict) and 'nbt' in json_body.keys():
                     self.current_json[key] = {}
                     self.current_json[key]["nbt"] = json_body["nbt"](arg)  # type: ignore # fmt: off
-                    self.current_json[key]["interpret"] = json_body["interpret"](arg)  # type: ignore # fmt: off
```

Comment:
This line here didn't even seem to actually have any effect, so I removed it for simplicity.